parse-dump.py

```diff
@@ -0,0 +1,60 @@
+import argparse
```

Comment:
this should live in the `scripts` directory. Also, please add instructions on how to use the script to the README.md (also in the `scripts` folder)